gms/api/src/main/pegasus/com/linkedin/glossary/GlossaryNodeKey.pdl

```diff
@@ -0,0 +1,20 @@
+namespace com.linkedin.glossary
+
+import com.linkedin.common.GlossaryNodeUrn
+
+/**
+ * Key for GlossaryNode resource
+ */
+record GlossaryNodeKey {
+
+  @validate.strlen = {
+    "max" : 20,
+    "min" : 1
+  }
+  name: string
+
+  /**
+   * Parent business nodes(in sequence) to this business term.
+   */
+  glossaryNodePath: string
```

Comment:
this would mean we would allow multiple glossary terms of the same name to exist within different paths. This is intentional, correct?

Reply:
Yes, example:
un:li:businessTerm:domain1.term
and urn:li:businessTerm:domain2.term can coexist.